posts/PartinGs Pvt 4 Gate Blink Allin.md

```diff
@@ -0,0 +1,42 @@
+<!--
+.. title: PartinG's PvT 4 Gate Blink All-In
+.. slug: PartinGs-PvT-4-Gate-Blink-Allin
+.. date: 2020-08-05
+.. tags: pvt, guide, beginner, early game
+.. category: Guides,videos
+.. link: https://www.youtube.com/watch?v=PmSrYtYGqdw
+.. description: Beginner-friendly PvT Semi-All-In
+.. type: text
+.. author: Harstem
+-->
+
+<iframe width="560" height="315" src="https://www.youtube.com/watch?v=PmSrYtYGqdw" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
For some weird reason this doesn't display in my Firefox properly... I'll take another look.